packages/legend-query-builder/style/_query-builder-post-filter.scss

```diff
@@ -164,6 +164,11 @@
     font-weight: 700;
     color: var(--color-dark-grey-300);
     height: 2.8rem;
+
+    &--potential__dropzone {
```

Comment:
--> `--dropable`

again, for these, we should think of a way to make them absolutely position to not jank the layout